src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -18,6 +18,11 @@
   --pf-c-context-selector__toggle--active--BorderBottomColor: var(--pf-global--active-color--100);
   --pf-c-context-selector__toggle--expanded--BorderBottomWidth: var(--pf-global--BorderWidth--md);
   --pf-c-context-selector__toggle--expanded--BorderBottomColor: var(--pf-global--active-color--100);
+  --pf-c-context-selector__toggle--m-plain--Color: var(--pf-global--Color--200);
+  --pf-c-context-selector__toggle--m-plain--hover--Color: var(--pf-global--Color--100);
+  --pf-c-context-selector__toggle--m-plain--disabled--Color: var(--pf-global--disabled-color--200);
+  --pf-c-context-selector__toggle--m-plain--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-context-selector__toggle--m-plain--PaddingLeft: var(--pf-global--spacer--sm);
```

Comment:
Let's use the `--md` spacer for these `--m-plain--Padding` vars. This will be used with `.pf-m-plain` on it's own. Then let's create 2 new vars for `--m-plain--m-text--Padding[Right/Left]` that use the `--sm` spacer.